src/adapters/postgres/cohortMembers-adapter.ts

```
@@ -1346,6 +1392,78 @@ export class PostgresCohortMembersService {
+        } else if (data.sourceDetails.source === 'table') {
+          const labels = await this.fieldsService.findDynamicOptions(
+            data.sourceDetails.table,
+            `value='${data.value}'`
+          );
+          if (labels && labels.length > 0) {
+            processedValue = labels[0].name;
+          }
+        }
+      }
+
+      delete data.fieldParams;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)